components/automate-ui/src/app/entities/data-bags/data-bag-details.effects.ts

```diff
@@ -51,7 +51,7 @@ export class DataBagItemsEffects {
       const msg = payload.error.error;
       return new CreateNotification({
         type: Type.error,
-        message: `Could not get infra data bag items: ${msg || payload.error}`
+        message: `Could not get data bag items: ${msg || payload.error}`
```

Comment:
Author note: consistency -- majority of uses omit the "infra" so several occurrences changed.